### PR TITLE
Flesh out tests

### PR DESCRIPTION
**Pull request by milofultz**

### CODE REVIEW

File: __tests__/_mockData.js [hunk 1]
```diff
@@ -82,12 +82,27 @@ module.exports = {
...
+      "recommend": true,
+      "reported": true,
       "reviewer_name": "Curtis_King46",
       "reviewer_email": "Dolly.Muller75@hotmail.com",
-      "response": "null",
+      "response": null,
       "helpfulness": 12
+    },
+    {
+      "_id": "60fad51d93cd1365f0ae7cad",
```
Comment (NylonEric) [inline __tests__/_mockData.js:93]: alpha-numeric identifier? did you implement some indexing or user id system?

File: __tests__/server.test.js [hunk 2]
```diff
@@ -90,6 +90,71 @@ describe('Server tests', () => {
...
+        recommend: false,
+        name: "Mister Twister",
+        email: "mister@twister.com",
+        photos: ["google.com"],
+        characteristics: {
+          "42": 4,
+          "41": 2
+        },
+      };
+      return axios.post('http://localhost:3000/reviews', body)
```
Comment (NylonEric) [inline __tests__/server.test.js:109]: Watch hard-coded addressing when you migrate to containers/deployment.

File: k6/getReviewsMeta.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import http from 'k6/http';
```
Comment (NylonEric) [inline k6/getReviewsMeta.js:1]: If you are trying for the last 10% of the data set for product_id, you should start around 900k. Not sure about other collections.

**Review by NylonEric — APPROVED** [review]
These all look good. I made one comment about picking your dataset. 

I'm not sure which of your queries involve which db collections but I know we're shooting for the last 10% of the data store to make the db search non-trivial ids/indexes.

File: __tests__/_mockData.js (hunk 1 above)
Comment (milofultz) [reply __tests__/_mockData.js:93]: This is just what mongo does. When I imported it, I probably should have made it the id's that are provided, but I didn't and here we are hahaha

File: __tests__/server.test.js (hunk 2 above)
Comment (milofultz) [reply __tests__/server.test.js:109]: good call!

File: k6/getReviewsMeta.js (hunk 3 above)
Comment (milofultz) [reply k6/getReviewsMeta.js:1]: Yeah I changed this later. Thank you!

**Pull request merged**